core/src/main/java/io/grpc/ServerImpl.java

```diff
@@ -249,6 +257,7 @@ public void transportTerminated() {
     @Override
     public ServerStreamListener streamCreated(final ServerStream stream, final String methodName,
         final Metadata.Headers headers) {
+      final Future<?> timeoutFuture = scheduleTimeout(stream, headers);
```

Comment:
Discussed offline, but this method just starts the Call. The method and Runnable can complete but the Call still be ongoing.